msi/GitProduct.wxs

```diff
@@ -115,15 +124,15 @@
     <Feature Id='Git'>
       <Component Directory='INSTALLFOLDER'>
         <RegistryValue Root='HKLM' Key='SOFTWARE\GitForWindows' Name='InstallFolder' Value='[INSTALLFOLDER]' Type='string' />
-        <Environment Name='PATH' Value='[BinFolder]' Action='set' Part='last' System='yes' />
         <RemoveFolder Directory='GitProgramMenuFolder' On='uninstall' />
         <RemoveFolder Id="RemoveApplicationFolder" Directory='ApplicationFolder' On='uninstall' />
       </Component>
       <Component Directory="INSTALLFOLDER">
         <RegistryValue Root="HKLM" Key="Software\GitForWindows" Name="CurrentVersion" Type="string" Value="$(var.Version)" />
       </Component>
-      <Component Directory='INSTALLFOLDER'>
-        <RegistryValue Action='write' Type='string' Root='HKLM' Key='SOFTWARE\GitForWindows' Name='CliEnvironment' Value='[CLIENVIRONMENT]' />
+      <Component Directory="INSTALLFOLDER">
+        <RegistryValue Action='write' Type='string' Root='HKLM' Key='SOFTWARE\GitForWindows' Name='CliEnvironment' Value='[CLIENVIRONMENT]' KeyPath="yes" />
+        <Environment Name="PATH" Value="[CLIENVIRONMENTPATHS]" Action="set" Part="last" System="yes" />
```

Comment:
`CLIENVIRONMENTPATHS` is used to hold all the paths to add to the `PATH` environment variable. If the user chooses to only use Git for Windows from the Bash command line, then `CLIENVIRONMENTPATHS` will be empty and, in effect, `PATH` will remain unmodified. Line 48 in this file ensures this:

```
<SetProperty Action="SetBashCliEnvironment"
    Id="CLIENVIORNMENTPATHS" Value="{}"
    Before="InstallValidate" Sequence="execute">
        CLIENVIRONMENT = "Bash"
</SetProperty>
```

The `Value="{}"` attribute is the MSI formatted-text type way of saying "empty string" for setting a property to an empty string. And as you can see from the inner text (which is a condition), this custom action runs when the user specifies to use Git for Windows only from the Bash shell.